Library/Formula/rebar3.rb

```diff
@@ -0,0 +1,19 @@
+class Rebar3 < Formula
+  desc "Erlang build tool"
+  homepage "https://github.com/rebar/rebar3"
+  url "https://github.com/rebar/rebar3/archive/3.0.0-beta.4.tar.gz"
```

Comment:
This will have to be [devel-only](https://github.com/Homebrew/homebrew-devel-only/) until there's a stable release.